src/printapp/templates/index.html

```diff
@@ -140,8 +153,14 @@ <h3 class="subheader">Welcome, {{email}}</h3>
             </thead>
             <tbody>
               {{#each item in controllers.queue}}
-                <tr>
-                  <td>{{item.name}}</td>
+                <tr {{bind-attr id=item.job_id}}>
```

Comment:
Remove this, then set attributes on the model such as "item.loading = true" and read that in the view to decide when to show the loading spinner.